src/resources/locales/en.json

```diff
@@ -63,7 +63,7 @@
     },
     "delete": "Delete",
     "deleteBook": "Delete",
-    "deleteTag": "Delete tag",
+    "deleteTag": "Deletße tag",
```

Comment:
typo, no?

Reply:
yeah, my new cheap keyboard :) , difficult to handle